internal/setup/setup.go

```diff
@@ -577,10 +592,14 @@ func parseRelease(baseDir, filePath string, data []byte) (*Release, error) {
 			Version:    details.Version,
 			Suites:     details.Suites,
 			Components: details.Components,
+			Pro:        details.Pro,
 			Priority:   details.Priority,
 			PubKeys:    archiveKeys,
 		}
 	}
+	if len(release.Archives) == 0 {
+		return nil, fmt.Errorf("%s: no valid archives defined", fileName)
+	}
```

Comment:
TODO: Should this be in `cmd_cut.go` only seeing how `info` and `find` may not need it?

Reply:
`slicer.go` might be a better idea.